[Mini guide] - Map Dynatrace entities to teams (#1986)
port-labs/port-docs: docs/guides/all/connect-dynatrace-team-with-entities.md

---
displayed_sidebar: null
description: Learn how to map Dynatrace teams to monitored entities in Port
---

# Assign teams to monitored entities

This guide explains how to assign team ownership to Dynatrace entities, allowing you to easily identify which team is responsible for each monitored entity.

## Common use cases
- Quickly determine which team owns a specific entity and contact them when needed.

## Prerequisites
This guide assumes the following:

> - You have a Port account and that you have finished the [onboarding process](/quickstart).

Reviewer: ```suggestion
- You have a Port account and you have completed the [onboarding process](/quickstart).
```

> - You have [installed and setup Port's Dynatrace integration](/build-your-software-catalog/sync-data-to-catalog/apm-alerting/dynatrace/dynatrace.md)

Reviewer: ```suggestion
- You have installed Port's [Dynatrace integration](/build-your-software-catalog/sync-data-to-catalog/apm-alerting/dynatrace/dynatrace.md).
```

- You have entities from cloud providers configured on Dynatrace. See [Dynatrace documentation](https://docs.dynatrace.com/managed#deploy-on) for this.

## Set up data model

> To establish team ownership,  we will modify the `Dynatrace Entity` blueprint by adding an `owned_by` relation. Follow the steps below to **update** the `Dynatrace Entity` blueprint:

Reviewer: ```suggestion
To establish team ownership,  we will modify the `Dynatrace Entity` blueprint by adding an `owned_by` relation.  
Follow the steps below to **update** the `Dynatrace Entity` blueprint:
```

> 1. Navigate to the `Dynatrace Entity` blueprint in your Port [Builder](https://app.getport.io/settings/data-model).

Reviewer: ```suggestion
1. Go to the [data model page](https://app.getport.io/settings/data-model) of your portal, and locate the`Dynatrace Entity` blueprint.
```

2. Hover over it, click on the `...` button on the right, and select "Edit JSON".

> 3. Add the `owned_by` relation as shown below:

Reviewer: ```suggestion
3. Add the `owned_by` relation as shown below, then click `Save`:
```

        - `cloud:azure:appplatform:spring`
        # see below section for more entity types
    port:
      entity:
        mappings:
          identifier: .displayName | gsub(" "; "-")
          title: .displayName
          blueprint: '"dynatraceEntity"'
          properties:
            firstSeen: .firstSeenTms / 1000 | todate
            lastSeen: .lastSeenTms / 1000 | todate
            type: .type
            tags: .tags[] | map(.stringRepresentation)
          relations:
            owned_by: .tags | map(select(.key == "dt.owner" or .key == "owner") | .value) | if length == 0 then null else . end
```
:::tip ownership keys
In this example, the `dt.owner` and `owner` keys from the tags are used to define ownership. You should use the keys configured in your Dynatrace environment. For more details on setting up ownership keys, refer to the [Dynatrace documentation](https://docs.dynatrace.com/docs/deliver/ownership/assign-ownership#format)
:::
</details>

Next, click on the **resync** button and watch your Dynatrace `entities` being mapped to the `teams` as shown below in this example:

<img src='/img/guides/dynatraceEntityTeamOwnership.png' border='1px' />

> By following this guide, you have successfully configured your Dynatrace integration to map monitored entities to their respective team owners.

Reviewer: No need for this last sentence
